helm/robusta/values.yaml

```diff
@@ -384,6 +384,8 @@ platformPlaybooks:
     - on_statefulset_all_changes: {}
     - on_replicaset_all_changes: {}
     - on_pod_all_changes: {}
+    - on_node_all_changes: {}
+    - on_job_all_changes: {}
```

Comment:
how noisy is this for jobs? 
like on one job that runs a few pods how often is this called?